### ISSUE TITLE
[HL2MP] Not Listed on Main Game

### ISSUE THREAD

**Issue by da11as911** [ValveSoftware/Source-1-Games#6527]
All other Half Life 2 installments (Ep1, Ep2, and even Lost Coast) are now listed on the main Half Life 2 game menu, why was HL2MP left off? Please add it so that players do not have to launch a separate game to find it, this is potentially game killing, especially now that Half-Life 2: Deathmatch is unlisted in the Steam Store.

**Comment by speedvoltage** [comment]
Looks as though the game is indeed missing in the store on my end.

**Comment by Lassebq** [comment]
It doesn't even work if you have it in library #6528

**Comment by doclic** [comment]
> It doesn't even work if you have it in library #6528

It does but only on Windows.

**Comment by henk717** [comment]
I recommend doing a shortcut to HL2DM instead where HL2 Singleplayer closes and asks steam to launch Half-Life 2 Deathmatch similar to how other games do it.
Couple advantages :
- You don't migrate us to yet another engine, so you don't break all the servers again.
- VAC does not get injected in the singleplayer game.
- Players who do not care about either singleplayer or multiplayer have a smaller download.
- Multiplayer still gets the promotion it deserves thanks to the redirect button.

**Comment by Tripperful** [comment]
In fact, hl2mp game folder along with some random server and client binaries is already present in the current HL2 distribution. You can even launch it by copying gameinfo.txt from your current hl2dm installation into Half-Life 2/hl2mp and running hl2 -game hl2mp. It's broken though, but even its existence (along with the fact hl2dm was unlisted from the store along with the other mods) confirms Valve's intention to eventually (hopefully soon) merge the games into one client distribution

![image](https://github.com/user-attachments/assets/29aa2e7e-258c-4cb9-ad7b-a2028bec66a4)
![image](https://github.com/user-attachments/assets/a3343b66-a3c7-466f-8e4d-f0b3c862678e)
![image](https://github.com/user-attachments/assets/0fd2cecc-10cd-4733-8ad3-ef20513adbd4)

**Comment by da11as911** [comment]
On second thought, this may be better if Valve just re-listed the game but made it free to play. Since it is now free with the HL2 Single Player, why not just give it a Free to Play listed page if you are unable to add it to the HL2 SP menu? Not to mention bringing players into the multi player experience who have never experienced the SP could bring them to purchase the game. Just my two cents. With it unlisted and not on the menu it's certain to die off. An announcement with making it free to play and listed would give a nice boost of players that we saw for the first few days when you made the  SP free for the weekend. Please strongly consider this. @kisak-valve

**Comment by da11as911** [comment]
Any update on this @kisak-valve ? -- the temporary boost of players is fading fast.

**Comment by henk717** [comment]
He only assigns / labels issues.

**Comment by da11as911** [comment]
If adding the multiplayer link to Half-Life 2 Single player is not an option then it is time to make HL2DM free to play and re-list it’s steam page with an announcement on the Steam Store Front. We are running out of time, player count is dropping. Please don’t kill it completely and give the community what they deserve, just like TF2. Thank you!

**Comment by da11as911** [comment]
This is still an issue and very costly to the community, a game in which is now de-listed has no "home base" except if someone buys Half-Life 2, and "stumbles" upon it in their steam library.  PLEASE FIX THIS ASAP. Give us a Steam Community page back with free to play and make a public announcement like you did with TF2, PLEASE.

**Comment by da11as911** [comment]
This is still an issue. PLEASE FIX THIS ASAP. Please re-list with free to play and make a public announcement like you did with TF2, PLEASE. Valve essentially hiding this game is killing it. Thank you.

**Comment by gtamike** [comment]
> This is still an issue. PLEASE FIX THIS ASAP. Please re-list with free to play and make a public announcement like you did with TF2, PLEASE. Valve essentially hiding this game is killing it. Thank you.

Re-listing Half-Life 2 Deathmatch as Free to play is a great idea.
If Valve can get Half-Life 2 Deathmatch in the Half-Life 2 Menu that would be a bonus.

**Comment by henk717** [comment]
Personally I wouldnt want it merged, keeping it seperate is nice but a call of duty style menu item that exits hl2 and launches hl2dm would be very welcome.

**Comment by gtamike** [comment]
Adding HL2 Deathmatch Menu in HL2 would be really good bringing in more players. After clicked it could simply close HL2 and HL2 Deathmatch loads up like it currently does now. That would be easy minimal effort from Valve to get done too.

Also if nothing gets done HL2 Deathmatch needs to be relisted on Steam.

**Comment by da11as911** [comment]
Yes a redirect from the regular Half-Life 2 menu would be great. Though, I think at this point the only thing that could potentially boost or hold a player base is going to be moving it to free to play with a public announcement. Also @gtamike I was able to get it relisted on Steam a while back, after a few emails to Valve.

**Comment by da11as911** [comment]
> Nah, the best solution is making a shortcut at HL2 main menu, and then if you click it and don't have HL2 Deathmatch installed in Steam, it begins being downloading and after download, it executes. This is minimal effort
> 
> Maybe someone could email Valve about this idea?

I have, their response was relisting the game on steam. I had this solution as well as just making the game free to play in general without the requirement of purchasing the SP. At this point, I think action should be taken on both of these ideas.

**Comment by bruhhhxyt** [comment]
Ye

**Comment by Xeogin** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/6527#issuecomment-3219452719

Since it contains HL2 assets, they'd have to remove them, which would require HL2 to be installed in order to pull them, effectively making things sort of the same as they are, but also enabling them to stop automatically adding it to players' libraries, which would be bad for visibility. It'd basically become an obscure mod at that point.

**Comment by henk717** [comment]
No this is not true. Half-Life 2 Deathmatch contains very few paid assets and those are exclusive to HL2DM. HL2 exclusive content are the singleplayer maps which hl2dn does not have.

If you use the default SDK 2015 repo and you compile the multiplayer you actually get a copy of Half-Life 2 Deathmatch runnable entirely free minus some maps. So there is no issue in them making the steam version free also.

The issue is that even though as a communuty we can legally distrubute a free HL2DM as a mod the discoverability of this solution is worse than people buying hl2 so its not a solution.

Having it free on steam like back in the ati / nvidia promo daya (This game was free for many years its why I started playing it 15 years ago) would help the game a lot.
